core/tests/behavior/read_only.rs

```diff
@@ -183,39 +181,9 @@ pub async fn test_read_only_read_full_with_special_chars(op: Operator) -> Result
     Ok(())
 }
 
-/// Read full content should match.
-pub async fn test_read_only_read_range(op: Operator) -> Result<()> {
-    let bs = op.range_read("normal_file", 1024..2048).await?;
```

Comment:
We should replace with `read_with` instead of removing.

Reply:
This review is not addressed.

Reply:
Sorry, I forgot to push my commit.

Reply:
Now, here it is.